stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+import '../../dist/feed-list.css';
+import '../../dist/avatar.css';
+
+export default {
+    title: 'Components/Feed List',
+    parameters: {
+        description: 'The feed list item control displays unformatted text accompanied by an optional user image.',

```

Comment:
The feed list displays feed list items that individually consist of a username, a written entry and an optional image. There is also a separate byline below that can contain a time stamp or an attribute in the form of free text.

##Usage
**Use the feed list if:**
- You expect multiple instances, such as notes or feed entries.
- The items in the feed list are homogenous.
- A user needs to input small amounts of text without formatting.

**Do not use feed list if:**
- The user needs to format the text (rich text editor).
- You need only a single text box instance. Instead, use **Text Area**.
- You want to display empty fields for the user to input feed entries. Instead, use **Feed Input**.